docs/changelog.rst

```diff
@@ -0,0 +1,23 @@
+Changelog
+=========
+
```

Comment:
I would remove this Changelog as it's not up to date with the CHANGELOG file

Reply:
Ah well, I copied the `ChangeLog` file and removed the, for me, hard to parse parts, or if the parts were only simple lines like "Travis.". In the end I decided only to include the latest changelog. 

If you think we should just add the whole file I'm fine with that.

Reply:
Works for me